### DISCUSSION TITLE
"Review Changes" button is not keyboard accessible

### DISCUSSION THREAD

**Discussion by lornajane** [community/community#101792] [Accessibility] [answered]
### Select Topic Area

Product Feedback

### Body

I noticed a regression (weeks ago, maybe early December) where the "review changes" button isn't accessible using [Vimium](https://vimium.github.io/). I'm a keyboard-only user and use vimium to focus and activate controls. What's really strange about it is that if I tab through all the elements to get to the button, I can focus and interact with it (or something nearby, I can't use developer tools enough to understand what's happening). But although Vimium sees three elements (two on the button, one on the dropdown), none of them can be focused or activated. Could someone take a look?

**Comment by lornajane** [comment, marked as answer]
This was bothering me for weeks but it seems to be fixed today, so I'll close this. Thanks!

**Reply by ghostinhershell** [reply]
Glad to hear it! I've gone ahead and marked your response here as the answer just in case anyone else has the same question and would like to reference this discussion post. Thanks so much for contributing in our Accessibility community 🚀
